tree/readspeed/src/ReadSpeed.cxx

```diff
@@ -331,7 +331,7 @@ Result ReadSpeed::EvalThroughputMT(const Data &d, unsigned nThreads)
 
    TStopwatch sw;
    sw.Start();
-   const auto totalByteData = pool.MapReduce(processFile, ROOT::TSeqUL{d.fFileNames.size()}, SumBytes);
+   const auto totalByteData = pool.MapReduce(processFile, ROOT::TSeqUL{(unsigned long)d.fFileNames.size()}, SumBytes);
```

Comment:
`std::vector::size` does not return the equivalent of an unsigned long on windows? :eyes: what does it return? what's `sizeof(std::size_t)` and `sizeof(unsigned long)` on Windows?

If the problem is actually that `std::size_t` is larger than `unsigned long`, then a simpler fix is probably:

```suggestion
   const auto totalByteData = pool.MapReduce(processFile, ROOT::TSeqUL(d.fFileNames.size(), SumBytes);
```

(`{}` do not allow narrowing conversions, `()` do)

Reply:
oh _come on_ windows: https://godbolt.org/z/T7GErrs4o :man_facepalming: